src/components/forms/Form/SubmitError.tsx

```diff
@@ -1,26 +1,33 @@
-import { ReactNode, useContext } from 'react';
+import { ReactNode, useContext, isValidElement, memo } from 'react';
 
 import { Alert, CubeAlertProps } from '../../content/Alert';
 
 import { FormContext } from './Form';
 
 type SubmitErrorProps = {
-  submitError?: ReactNode;
+  submitError?: unknown;
 };
 
 /**
  * An alert that shows a form error message received from the onSubmit callback.
  */
-export function SubmitError(props: CubeAlertProps) {
-  const { submitError } = useContext(FormContext) as SubmitErrorProps;
+export const SubmitError = memo(function SubmitError(props: CubeAlertProps) {
```

Comment:
It doesn't make sense to memoize this component since it accepts props from an unknown environment which makes it very easy to break the memoization